src/formats/mmCIF.cpp

```diff
@@ -83,41 +83,42 @@ mmCIFFormat::mmCIFFormat(std::string path, File::Mode mode, File::Compression co
         }
 
         if (line_split[0] == "_cell_length_a" || line_split[0] == "_cell.length_a") {
-            a = cif_to_double(line_split[1]);
+            a = cif_to_double(line_split[1].to_string());
```

Comment:
Maybe `cif_to_double` should take a `string_view` argument.

Reply:
it can't, since it needs to erase the potential `(` and `)`.